spec/lrama/parser_spec.rb

```diff
@@ -42,6 +42,64 @@
     HEADER
   end
 
+  describe "#initialize" do
+    context 'when debug is true and define is not include `parse.trace`' do
+      context 'with define is not specified' do
+        it '@yydebug is true' do
+          parser = Lrama::Parser.new("", "", true)
+          yydebug = parser.instance_variable_get(:@yydebug)
+          expect(yydebug).to be_truthy
+        end
+      end
+
+      context 'with define is specified' do
+        context 'with `api.pure`' do
+          it '@yydebug is false' do
+            parser = Lrama::Parser.new("", "", true, {'api.pure' => nil})
+            yydebug = parser.instance_variable_get(:@yydebug)
+            expect(yydebug).to be_truthy
```

Comment:
This expectation conflicts with the description on line 57.

Reply:
Oops, I fixed it.